content/200-concepts/300-more/100-editor-setup.mdx

```diff
@@ -2,6 +2,7 @@
 title: 'Editor setup'
 metaTitle: 'Editor and IDE setup (Reference)'
 metaDescription: 'Learn how to configure your editor and IDEs for an optimal developer experience with Prisma.'
+tocDepth: 2
 ---
 
 ## Overview
```

Comment:
@molebox Since you're working on this - could you replace Overview with `<TopBlock>` around everything before the first h2?